scripts/release/debian/deb_install.sh

```diff
@@ -81,7 +92,7 @@ setup() {
             exit 1
         fi
     fi
-    echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ ${CLI_REPO} main" \
+    echo "deb [arch=amd64 signed-by=/etc/apt/keyrings/microsoft.gpg] https://packages.microsoft.com/repos/azure-cli ${CLI_REPO} main" \
```

Comment:
How about using `` arch=`dpkg --print-architecture` ``? We plan to support arm64.

Reply:
@bebound , great news about the arm64 support! I have added your suggested change.
 
**I have also added in some additional logic to remove the deprecated key from the main keyring if found.** This will avoid duplicates on any system where they've run the older script and then run this one.